docs/catalog/cloud/encrypt-what-is-necessary

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: @amexboy @gopaltayal @samudurand @shainu-x
```

Comment:
I'm not a 100% sure which code evaluates this (if any), but all other patterns do not have the @ and separate by commas, instead of spaces (IIRC).